src/frontend/src/btc/services/btc-address.services.ts

```diff
@@ -0,0 +1,121 @@
+// We use the Testnet address for Regtest.
```

Comment:
This comment seems to be outdated, could you please remove it since you're doing changes here?

Reply:
done